File: LeetCode/2011/Solution.java
class Solution {
    public int finalValueAfterOperations(String[] operations) {
        int result = 0;
        for(String operation: operations) {
            if(operation.charAt(0) == '-' || operation.charAt(2) == '-') {
                result--;
            } else {
                result++;
            }
        }
        return result;
    }
}

File: LeetCode/2011/Solution.kt
class Solution {
    fun finalValueAfterOperations(operations: Array<String>): Int {
        var ret = 0
        for(operation in operations) {
            if(operation.equals("--X") || operation.equals("X--")) {
                ret--
            } else {
                ret++
            }
        }
        return ret
    }
}
